index.js

```diff
@@ -22,4 +25,13 @@ app.use("/user", userRouter);
 app.use("/restaurant", restaurantRouter);
 app.use("/tables", tableRouter);
 
+httpServer.listen(80);
```

Comment:
why did you use port 80 ?

Reply:
I misunderstood the initialization of socket io, now it is the same port specified for app in env file.